df.history.xml

```diff
@@ -809,21 +837,24 @@
         <int32_t name='race' ref-target='creature_raw'/>
         <int16_t name='caste' ref-target='caste_raw' aux-value='$$.race'/>
 
-        <int32_t/>
-        <int32_t name='histfig_id' ref-target='historical_figure'
-                 comment='when masquerading as another historical figure'/>
-        <int32_t name='unk_4c'/>
+        <int32_t name='unk_1'/>
+        <compound is-union='true' union-tag-attribute='tag'>
```

Comment:
I hadn't thought of getting rid of the union name, but I like the result!

Reply:
Update: it's just `union-tag-attr`, not `-attribute`. Travis in this repo was broken, so that didn't get reported back, but I'm migrating to GitHub actions as we speak, which will hopefully be nicer to deal with.